dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -106,6 +106,38 @@ abstract class DLCWallet extends Wallet with AnyDLCHDWalletApi {
     } yield updated
   }
 
+  private def updateFundingOutPoint(
```

Comment:
nit: Can you add a scaladoc? What is it updating? It looks like it updates `DLCDb` adding the outpoint to the database?